Assignment-07-Product Designer/styles/css/assignment7.css

```diff
@@ -0,0 +1,150 @@
+body {
+  background-color: black;
+  margin: 0;
+  padding: 0;
+}
+
+.initial_header {
+  margin-top: 40px;
+  display: flex;
+  flex-direction: row-reverse;
+  color: #ffff;
+  margin-bottom: -8px;
+  font-family: "Roboto", "sans-serif";
+  font-weight: 600;
+  width: 88.4%;
+}
+
+.flex_container_shrutti {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  color: #ffff;
+  margin-top: 60px;
+}
+
+.shrutti {
+  font-size: 60px;
+  font-weight: 700;
+  font-family: "Roboto", "sans-serif";
```

Comment:
Why to declare font name when it is declared in body ?

Reply:
font name : 'Roboto','sans-serif' made public to using in the body tag.